templates/chrony.conf

```diff
@@ -0,0 +1,43 @@
+{% if chrony_servers is defined -%}
+
```

Comment:
Can you delete this blank line as well as the one on L6 to make it consistent with the rest of the file?